quick_start/more.rst

```diff
@@ -0,0 +1,163 @@
+Field Name        Description              Type
+================  =======================  ===========
+``species``       The species of iris      Text
+
+``sepal_width``   The width of the sepal   Double
+
+``sepal_length``  The length of the sepal  Double
+
+``petal_width``   The width of the petal   Double
+
+``petal_length``  The length of the petal  Double
+================  =======================  ===========
+
+If we are trying to find the species based on the sepal and petal measurements, this means these measurements are going to be the input to our classifier module, with text being the output. This means we need to write a method in Python which takes four ``Double``\s and returns ``Text``.
+
+Creating your classifier module
+******************************
+
+To begin, let's make a new directory called ``irisclassify``, ``cd`` into it, and initialise a new module:
+
+.. code:: bash
+    
+    ~/ $ mkdir irisclassify; cd irisclassify
+    ~/irisclassify/ $ nstack init python
+    python module 'irisclassify' successfully initialised at ~/irisclassify
+
+Next, let's download our training data into this so we can use it in our module.
+
+.. code:: bash
+
+    ~/irisclassify/ $ curl -O https://raw.githubusercontent.com/nstackcom/nstack-examples/master/iris/irisclassify/train.csv
+
+
+Defining our API
+****************
+
+As we know what the input and output of our classifier is going to look like, let's edit the ``api`` section of ``nstack.yaml`` to define our API (i.e. the entry-point into our module). By default a new module contains a sample method ``numChars``, which we can replace with our definition. We're going to call the method we write in Python ``predict``, which means we can fill in the ``api`` section of ``nstack.yaml`` as follows:
+
+.. code :: java
+
+    api : |
+        interface Default {
+          predict : (Double, Double, Double, Double) -> Text
+        }
+
+This means we are exposing a single method ``predict``, which takes a record of four ``Double``\s (the measurements) and returns ``Text`` (the iris species).
+
+
+Writing our classifier
+**********************
+ 
+Now that we've defined our API, let's jump into our Python module, which lives in ``service.py``.
+We see that NStack has created a class ``Service``. This is where we add the methods for our module. Right now it also has a sample method in it, ``numChars``, which we can remove. 
+
+
+Firstly, let's import the libaries we're using.
+
+.. code :: python
+
+    import nstack
+    import pandas as pd
+
+    from sklearn.ensemble import RandomForestClassifier
+
+.. note :: Python modules must also import ``nstack``
+
+Before we add our ``predict`` method, we're going to add ``__init__``, the Python contructor method which runs upon the creation of our module. It's going to load our data from ``train.csv``, and use it to train our Random Forest classifier:
+
+.. code :: python
+
+    def __init__(self):
+        train = pd.read_csv("train.csv")
```

Comment:
Even our examples escape our type system!

Not a criticism as it seems like its actually the main use case